### ISSUE TITLE
Is there anyway to calculate the mAP with custom objects? I have 8 classes and am using VOC standard.

### ISSUE THREAD

**Issue by rosyapril** [AlexeyAB/darknet#2041]
Did not find the methods in the thread. Thanks!

**Comment by AlexeyAB** [comment]
Yes, use this command in this repository: https://github.com/AlexeyAB/darknet#when-should-i-stop-training
`./darknet detector map data/obj.data yolo-obj.cfg backup\yolo-obj_7000.weights`
